### PR TITLE
Pull in latest Unicode 14 ucd from Public/

### PR DESCRIPTION
**Pull request by Manishearth**
Generated files posted at https://corp.unicode.org/~book/incoming/manishearth/u14/d8/

cc @markusicu @Ken-Whistler

### CODE REVIEW

**Review by markusicu — DISMISSED** [review]
input data changes lgtm

haven't looked at output

**Comment by markusicu** [comment]
I am comparing the output files with what's currently posted.

The generated PropertyValueAliases.txt re-removes the new script codes, so that probably needs more code changes (additions to hardcoded enums). Please check the instructions. If it's unclear, we can look together, dig in the change history, etc. (Sorry, I do this only once a year and the details elude me.)

The generated file does add the age=14.0 value, so that's good.

**Comment by markusicu** [comment]
The other generated files look at least plausible.

**Comment by Manishearth** [comment]
@markusicu Pushed fixes, posted at same URL

**Comment by Manishearth** [comment]
Unfortunately the built-in diff only prints the first line of the diff, which is often broken for other reasons (copyright, or in this case the age enum)

**Comment by Manishearth** [comment]
What had happened was that there were still new scripts to be added from the latest data but I  didn't get compile errors so I didn't notice. I updated some of the instructions.

File: unicodetools/org/unicode/props/UcdPropertyValues.java [hunk 1]
```diff
@@ -1551,8 +1557,10 @@ public static Numeric_Type_Values forName(String name) {
         Thai("Thai"),
         Tibetan("Tibt"),
         Tirhuta("Tirh"),
+        Tangsa("Tnsa"),
```
Comment (markusicu) [inline unicodetools/org/unicode/props/UcdPropertyValues.java:1560]: Looks like after this line we should also have
```
Toto("Toto"),
```

**Review by markusicu — COMMENTED** [review]
Thanks, but there is a bug in the output -- two of the script values have their long names swapped:
```
...
sc ; Ougr                             ; Tangsa
...
sc ; Tnsa                             ; Old_Uyghur
...
```
There is apparently a Toto line missing in the code (see below) but it does not seem like that should cause swapping of these two values. And I don't see where else your additions are out of sync :-/

Sometimes I have gotten tripped up by an optimization that Mark has built into his tools: The tools write BIN and .bin files with serialized property values (serialized UnicodeMaps). I always try to remember and delete these for the version of Unicode I am working on before I run the tools. Otherwise the tools may read the old .bin files, which in this case might have Toto=159 although you are changing it to 162. Also try to "clean all projects" in your Eclipse workspace.

**Comment by Manishearth** [comment]
> Thanks, but there is a bug in the output -- two of the script values have their long names swapped:

Oh! I did that on purpose for testing and forgot to undo it. Hold on, I'll fix it

**Comment by markusicu** [comment]
> What had happened was that there were still new scripts to be added from the latest data but I didn't get compile errors so I didn't notice. I updated some of the instructions.

Hey, no need to apologize! I always have to fumble for a while in these tools (since I also only work on them once a year for a few discontiguous days) and do trial & error.

It's always good to check the output. And I just figured out how to ignore the "Date:" line in Meld :-)

**Comment by Manishearth** [comment]
> It's always good to check the output. And I just figured out how to ignore the "Date:" line in Meld :-)

Oh, interesting, how? I've never used Meld to just diff directories

**Comment by Manishearth** [comment]
Updated, and posted latest files in the same spot

**Comment by markusicu** [comment]
> > It's always good to check the output. And I just figured out how to ignore the "Date:" line in Meld :-)
> 
> Oh, interesting, how? I've never used Meld to just diff directories

Meld > Preferences > Text Filters > "+" to Add new filter > name=`unidata Date` pattern=`^# Date: .+$`

**Comment by Manishearth** [comment]
Oh, that's neat.

Seems like the only common files between generated and original are Scripts and PropertyValueAliases.txt, is that expected? The diff does look okay now.

**Review by markusicu — APPROVED** [review]
lgtm pse squash & merge

**Comment by markusicu** [comment]
Thanks, confirmed, output is fixed.

**Comment by markusicu** [comment]
> Seems like the only common files between generated and original are Scripts and PropertyValueAliases.txt, is that expected?

I think so, until more files are posted.
I have manually created a folder with Unicode 13 files where 14 files are missing, and am diffing that collection with your output.

**Pull request merged**